sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/CheckpointManager.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs;
+
+import com.azure.messaging.eventhubs.eventprocessor.models.Checkpoint;
+import com.azure.messaging.eventhubs.eventprocessor.models.PartitionContext;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * The checkpoint manager that clients should use to update checkpoints to track progress
```

Comment:
Consistently use periods at the end of your statements. You use them in some and not others.